test/PackageManager/API/PackageDeploymentManagerTests_IsReady.cpp

```diff
@@ -124,86 +130,148 @@ namespace Test::PackageManager::Tests
             RemovePackage_Redder();
         }
 
+        TEST_METHOD(IsPackageReady_PackageFamilyName_InvalidParameter)
+        {
+            auto packageDeploymentManager{ winrt::Microsoft::Windows::Management::Deployment::PackageDeploymentManager::GetDefault() };
+
+            try
+            {
+                PCWSTR c_packageFamilyName{ L"Does.Not.Exist_1234567890abc" };
+                packageDeploymentManager.IsPackageReady(c_packageFamilyName);
+                VERIFY_FAIL(L"Success is not expected");
+            }
+            catch (winrt::hresult_error& e)
+            {
+                VERIFY_ARE_EQUAL(E_INVALIDARG, e.code(), WEX::Common::String().Format(L"0x%X %s", e.code(), e.message().c_str()));
+            }
+        }
+
         TEST_METHOD(IsPackageSetReady_InvalidParameter)
         {
+WEX::Logging::Log::Comment(L"P1");
```

Comment:
Looks like you left out the debug printfs here

Reply:
Good catch. Thanks